HW5/HW5.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import (MultipleLocator, FormatStrFormatter, AutoMinorLocator)
+
+def read_gff(gff_annotation_file_name):
+    gff_annotation = pd.read_csv(gff_annotation_file_name, sep='\t', skiprows=1, header=None, names=['seq_id', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
+    return gff_annotation
+read_gff('rrna_annotation.gff')

```

Comment:
Стоит отделять функции от кода вне переносами строк
```suggestion
def read_gff(gff_annotation_file_name):
    gff_annotation = pd.read_csv(gff_annotation_file_name, sep='\t', skiprows=1, header=None, names=['seq_id', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
    return gff_annotation
  
read_gff('rrna_annotation.gff')

```